### PR TITLE
[one-cmds] Document for how to create hdf5 dataset

### PR DESCRIPTION
**Pull request by jinevening**
Add document for how to create hdf5 dataset

ONE-DCO-1.0-Signed-off-by: Hyukjin Jeong <hj1.jeong@samsung.com>

---
Related to: https://github.com/Samsung/ONE/issues/4944

### CODE REVIEW

File: compiler/one-cmds/how-to-create-hdf5-dataset.txt [hunk 1]
```diff
@@ -0,0 +1,96 @@
...
+├── bald-eagle.jpg
+├── cow.jpg
+├── deer-in-wild.jpg
+├── fox.jpg
+├── ladybird.jpg
+├── orange-portocaliu.jpg
+├── pink-lotus.jpg
+├── red-church.jpg
+├── tomatoes.jpg
+└── young-dachshund.jpg
```
Comment (seanshpark) [inline compiler/one-cmds/how-to-create-hdf5-dataset.txt:46]: Is there no problem with copyrights with the image?

File: compiler/one-cmds/how-to-create-hdf5-dataset.txt [hunk 2]
```diff
@@ -0,0 +1,96 @@
...
+
+$ pip install Pillow numpy
+
+Run the pre-processing logic for the target model.
+We provide a short python script that scales the image data from -1 to 1.
+(This is different from the original pre-processing of InceptionV3.
+Visit the below link to find the exact algorithm)
+https://github.com/tensorflow/models/blob/v2.3.0/research/slim/preprocessing/inception_preprocessing.py
+
+$ cat > preprocess.py << EOF
```
Comment (seanshpark) [inline compiler/one-cmds/how-to-create-hdf5-dataset.txt:60]: maybe add "\`\`\`py" to make this as code block ?

File: compiler/one-cmds/how-to-create-hdf5-dataset.txt (hunk 1 above)
Comment (jinevening) [reply compiler/one-cmds/how-to-create-hdf5-dataset.txt:46]: Yes. I downloaded them from [freeimages.com](https://freeimages.com). The photos are free if we do not use them for resale. Here is the full [content lincense agreement](https://www.freeimages.com/photo/eagle-1523807).

**Review by seanshpark — APPROVED** [review]
LGTM

**Pull request merged**